bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -174,9 +176,11 @@ parser = argparse.ArgumentParser(usage="",
 
 pycbc.add_common_pycbc_options(parser)
 parser.add_argument("--version", action=pycbc.version.Version)
-parser.add_argument("--template-fit-file", required=True,
-                    help="hdf5 file containing fit coefficients for each"
-                         " individual template. Required")
+parser.add_argument("--template-fit-file", required=True, nargs='+',
+                    help="hdf5 file(s) containing fit coefficients for each "
+                         "individual template. Can smooth over multiple "
+                         "files provided they correspond to the same bank "
+                         "and fitting settings. Required")
```

Comment:
What happens if you give multiple files with different banks?  

Same question, for multiple files with the same bank but different fitting methods?

Reply:
I guess it's also true that all the files have to be from the same ifo?  (Is this necessary to be hard-coded as impossible, though?)

Reply:
Different banks will not actually error - I thought that I had added that check, so I will add that.

As the bank file is needed to get the parameters, it doesn't make sense to use files from different banks, unless I make it so we can use multiple bank files as well, but that could become complicated.

Errors will be raised (during attribute checks) if ifo, fitting methods, or stat threshold are different. The latter two I think would make the averaging nonsensical, but different IFOs should be allowed, like you say the attributes will become difficult